client/components/download-header.js

```diff
@@ -1,80 +0,0 @@
-import React, { useState, useRef, useEffect, Fragment } from 'react';
-import { useSelector, shallowEqual } from 'react-redux';
-import DownloadLink from './download-link';
-
-const selector = (({
-  project: { title, id },
-  application: {
-    isGranted,
-    basename,
-    drafting
-  }
-}) => ({
-  id,
-  title,
-  isGranted,
-  basename,
-  drafting
-}))
-
-export default function DownloadHeader() {
-  const [modalShowing, updateModalShowing] = useState(false);
-  const container = useRef(null);
-  const download = useRef(null);
-
-  const props = useSelector(selector, shallowEqual);
-  const {
-    id,
-    title,
-    isGranted,
-    drafting,
-  } = props;
-  let { basename = window.location.href } = props;
-
-  basename = basename.replace(/\/edit/, '');
```

Comment:
I think this is missing from the asl/components version - I guess we need it?

Reply:
Sorted here https://github.com/UKHomeOffice/asl-projects/pull/367/files#diff-aa416c553720e93e8852480dcca1b545R99